OlinkAnalyze/vignettes/bridging_E3072toEHT.Rmd

```diff
@@ -0,0 +1,682 @@
+
+## Bridge Sample Selection
+
+Prior to running a study with Explore HT, bridging samples must be selected
+from the study run with Explore 3072 and be run on the Explore HT study. These samples can
+be selected using the `olink_bridgeselector()` function in Olink Analyze
+as detailed in [the Introduction to bridging
+tutorial](https://cran.r-project.org/web/packages/OlinkAnalyze/vignettes/bridging_introduction.html#selecting-bridging-samples). The recommended number of bridge samples for within- and between- product bridging is summarized in the table below. When selecting bridge samples, the aim is to select samples that represent the dynamic range of the assay expression in the product. As such, quality control of the sample and, if available, proportion of data above LOD in the sample are considered when determining if a sample is chosen as a bridging sample. When LOD data is not available in the data expert from Olink NPX software, LOD can optionally be calculated from fixed LOD or negative controls as
+detailed in the [Calculating LOD from Olink Explore data
+tutorial](https://cran.r-project.org/web/packages/OlinkAnalyze/vignettes/LOD.html).
+
+```{r brnrtab, eval=TRUE, message=FALSE}
+data.frame(Platform = c("Target 96",
+                         "Explore 384 Cardiometabolic, Inflammation, Neurology, and Oncology",
+                        "Explore 384 Cardiometabolic II, Inflammation II, Neurology II, and Oncology II",
+                        "Explore HT", 
+                        "Explore 3072 to Explore HT"),
+           BridgingSamples = c("8-16",
+                               "8-16",
+                              "16-24",
+                              "16-32",
+                              "40-60")) |>
+  kbl(booktabs = TRUE,
+      digits = 2,
+      caption = "Table 1. Recommended number of bridging samples for Olink platforms") |>
+  kable_styling(bootstrap_options = "striped",
+                full_width = FALSE,
+                position = "center",
+                latex_options = "HOLD_position")
+
+```
+
+
+
+## Workflow Overview
+
+Olink Explore 3072 to Olink Explore HT bridging requires Explore 3072
+data and Explore HT data which have at least 40 to 64 bridging samples. For
+studies containing multiple projects of Explore 3072 data, the Explore
+3072 data sets should be bridged using within-product bridging as
+detailed in the [Introduction to bridging
+tutorial](https://cran.r-project.org/web/packages/OlinkAnalyze/vignettes/bridging_introduction.html) or otherwise normalized together
+prior to performing between-product bridging.
+
+The assays from Explore 3072 are matched to the corresponding assays in
+Explore HT and evaluated to determine if the assay is bridgeable. Additionally, all assays are normalized using both quantile smoothing and
+normalization using the median of paired differences. The result is an
+adjusted Explore 3072 data set with three additional columns:
```

Comment:
Should this reflect Table 4 in the Function Output section? Seems like there are 5 additional columns:

1. OlinkIDE3072 (OlinkID will reflect the HT ID)
2. MedianCenteredNPX
3. QSNormalizedNPX
4. Project
5. BridgingRecommendation

Reply:
Function Output section also mentions an OlinkID_HT column that's added to the output.